### ISSUE TITLE
Compatible With WPML

### ISSUE THREAD

**Issue by Softsell20** [wpexpertsio/license-manager-for-woocommerce#195]
Problem*
When running WPML the plugin sees each language as a different product
Option*
Would be nice to have an option to connect the licenses to all the different languages

**Comment by jadolyo** [comment]
I agree with this thread, WPML plugin creates a different page with a new id so it will be considered a different product, So we need a method to sync the stock with all different languages, I can't translate my products to another language and I am using fallback in the WPML plugin so I think it will be great for SEO and so on to have different language for the same products with synced stock.
